doc/conf.py

```diff
@@ -416,20 +416,49 @@ def get_test_id_list():
         )
     return rst_list
 
+def get_analysis_list(prefix):
+    from lisa.analysis.base import TraceAnalysisBase
+    from lisa.utils import get_subclasses
+
+    rst_list = []
+
+    for subclass in get_subclasses(TraceAnalysisBase):
+        class_path = "{}.{}".format(subclass.__module__, subclass.__qualname__)
+        analysis = subclass.__module__.split(".")[-1]
+        meth_list = [name for name, member in inspect.getmembers(subclass, callable)
+                     if name.startswith(prefix)]
+
+        rst_list += [":class:`{0}<{1}>`::meth:`~{1}.{2}`".format(analysis, class_path, meth)
```

Comment:
can we use named placeholders instead of positionals ? That makes the template more readable:
```
         rst_list += [":class:`{analysis}<{cls_path}>`::meth:`~{cls_path}.{meth}`".format(analysis=analysis, cls_path=class_path, meth=meth) 
```
```

Reply:
Hmm after giving it a go I find it makes the template less readable actually. Can I keep it as is?

Reply:
It's not very straightforward to check that the template is sticking the right things together, but if it is for you then it's ok. For example sticking {analysis}.{meth} would not be correct but it's not so obvious when we only have numeric placeholders.